src/matrix.js

```diff
@@ -105,22 +228,38 @@ SVG.Matrix = SVG.invent({
         y = x
       }
 
-      return this.around(cx, cy, new SVG.Matrix(x, 0, 0, y, 0, 0))
+      // Rotate the current matrix
```

Comment:
Should be "Scale the current matrix" I think

Reply:
good catch!